File: Guardiapp/app/src/main/java/com/intermodular/guardiapp/model/Horario.kt
package com.intermodular.guardiapp.model

import com.google.gson.annotations.SerializedName

data class Horario (

    @SerializedName("id")
    public val id: Int,
    @SerializedName("profesor")
    public val profesor: Int,
    @SerializedName("dia_semana")
    public val dia_semana: Int,
    @SerializedName("hora")
    public val hora: Int,
    @SerializedName("aula")
    public val aula: String?,
    @SerializedName("grupo")
    public val grupo : String?,
    @SerializedName("materia")
    public val materia : String?,
    @SerializedName("genera_guardia")
    public val genera_guardia : Boolean

        ): java.io.Serializable

File: GuardiApp/GuardiApp/Horario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GuardiApp
{
    public class Horario
    {
        public int id { get; set; }
        public int profesor { get; set; }   
        public int dia_semana { get; set; }
        public int hora { get; set; }
        public string aula { get; set; }
        public string grupo { get; set; }
        public string materia { get; set; }
        public bool genera_guardia { get; set; }
    }
}
